doc/resync_broken_replica.md

```diff
@@ -0,0 +1,18 @@
+# Re-sync a broken replica
```

Comment:
To keep naming consistent I'll rename it to Reset/Resync a broken keeper

Reply:
@sgotti I use the term "replica" because one cannot resync _any_ keeper. The master has to be intact for this to work.

Reply:
What about `Reset/Resync a broken standby keeper`?

Reply:
Will incorporate this suggestion.